src/main/java/Alex.java

```diff
@@ -0,0 +1,79 @@
+import java.util.Scanner;
+
+public class Alex {
+    public static void main(String[] args) {
+        Task[] allTasks = new Task[100];
+        int numberOfTasks = 0;
+        Scanner myObj = new Scanner(System.in);  // Create a Scanner object
+        greeting();
+        boolean isBye = false;
+        while(!isBye){
+            String userInput = myObj.nextLine();
+            String[] words = userInput.split(" ");
+            if(words[0].toLowerCase().equals("list")){
+                for(int i = 1; i <= numberOfTasks; i++){
+                    System.out.print(i);
+                    System.out.println(". " + "[" + allTasks[i-1].getStatusIcon() + "]" + " " + allTasks[i-1].getDescription());
```

Comment:
This code appears in very similar forms below. Perhaps it would be a good idea to extract this out to be a method of `Task`?